stix_shifter_modules/cbcloud/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
we found another important issue. two required stix objects are missing in the mapping `first_observed` and `last_observed`. I found few timestamp field in the results such as `backend_timestamp`, `device_timestamp` and `process_start_time`. one of them may needs to be mapped to those fields.